package-lock.json

```diff
@@ -11,8 +11,11 @@
         "@testing-library/jest-dom": "^5.16.4",
         "@testing-library/react": "^13.3.0",
         "@testing-library/user-event": "^13.5.0",
+        "bootstrap": "^5.2.0-beta1",
```

Comment:
remove package-lock.json file and add it to .gitignor, commit and push and then repeat npm install command